easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.6.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,31 @@
+name = 'TensorFlow'
+version = '1.6.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
```

Comment:
@smoors Actually, no, seems like you forgot to include this patch @smoors?

Why not use the existing patch for TF 1.5.0 which still applies, which is what we also did in #5951.

Reply:
Ah that patch is actually part of PR #6016. It is the same as the 1.5.0 version but with different line numbers. I do not know if eb is able to handle this.

Reply:
@smoors As long as `patch` is able to apply the patch file (it's smart enough to deal with slightly shifted patches), EB is happy too.